src/aap_eda/api/views/activation.py

```diff
@@ -652,7 +681,7 @@ def filter_queryset(self, queryset):
         return super().filter_queryset(queryset)
 
     @extend_schema(
-        description="List all logs for the Activation Instance",
+        description="List logs for an activation instance by ID",
```

Comment:
Nitpick: the descriptions of other endpoints that do something on an activation by ID do not write that in the description (whereas it fits into the newly added `x-ai-description`).